tests/lib/yaml_test.py

```diff
@@ -252,6 +252,18 @@ def testTypeErrors(self):
             "Filter must contain a terms section. File=policy_no_terms.pol.yaml, Line=3.",
         )
 
+        with self.assertRaises(yaml_frontend.PolicyTypeError) as arcm:
+            yaml_frontend.ParsePolicy(
+                IGNORED_YAML_POLICY_NO_TERMS,
+                filename="policy_no_targets.pol.yaml",
```

Comment:
do we want to remove `.pol.yaml` and use just `.yaml`

Reply:
It looks like this file contains a few dozen references to ".pol.yaml". Probably better to call this out of scope and do a follow up PR to avoid making this diff any larger.

Reply:
sgtm